contrib/test/integration/ansible.cfg

```diff
@@ -298,6 +292,7 @@ ssh_args = -o ControlMaster=auto -o ControlPersist=60s -o UserKnownHostsFile=/de
 # Example:
 # control_path = %(directory)s/%%h-%%r
 #control_path = %(directory)s/ansible-ssh-%%h-%%p-%%r
+control_path = /tmp/crio-%%n-%%p
```

Comment:
Not clear why you need to manually change this one

Reply:
Ahh, that's good feedback.  It's an ugly gotcha for Ansible-in-CI that will absolutely KILL performance.  I explained it in the commit-message, but it really should be in a comment also.  Thanks, I'll add that.